service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -22,13 +22,7 @@
 import io.opencensus.contrib.spring.aop.Traced;
 import io.opencensus.trace.Tracing;
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
nit: avoid * imports. there's a setting somewhere in intelliJ to stop it from replacing import lists with these

Reply:
Thanks, fixed. I updated the intelliJ settings for this but I guess made this change before that update.